.github/workflows/node.js.yml

```diff
@@ -28,9 +26,12 @@ jobs:
         cache: 'npm'
     - run: npm i -g npm@7
     - run: npm ci
+    - run: npm ci --prefix=test-project
     - run: npm run build --if-present
     - run: npm test
     - run: npm run lint
+    # throw error if lint has no error
+    - run: (npm run lint --prefix=test-project && exit 1) || exit 0
```

Comment:
This is not needed because the unit test (builder.spec.ts) already tests the functionality with the latest stable @angular-devkit version.
 
If we want to make it an integration tests with other angular version (multi version tests) we have to find another way. We would need to dynamically link the package in the test for every version to be tested.

Reply:
what about running the lint command on windows, macos and linux via matrix in the pipeline?

Reply:
good idea because maybe the issue in issue #94 can be reproduced and fixed(path escaped correctly) within this library

Reply:
it is actual possible with one of my projects called handpick... we could defined custom dependencies like ng12Dependencies inside pqckage.json and install them inside the CI

https://www.npmjs.com/package/handpick

Reply:
@krema that wasy idea

Reply:
Sorry for the delay. I couldn't find much time but made it. There is now a new Github action that tests against multiple Angular versions. I would also like to make this library backward compatible to Angular version 9.